main.go

```diff
@@ -9,14 +9,12 @@ import (
 
 func main() {
 	var debug bool
-
```

Comment:
How did removing blank lines force a new build? This can be squashed in, just don't understand the commit message.

Reply:
Dumb me. I didn't know if Drone was keeping something in memory (from a docker build or something like that), I had a problem with that on an old company that we used GitlabCi,  After some flaky tests I decided to do that for testing.  

I was wrong, the flaky was being caused by a race condition that was solved after adding a print to debug it :/.

Reply:
No worries, understood that was with Drone. For visibility, we have an open tf issue https://github.com/rancher/rancher/issues/42212 to deal with flaky tests so hopefully those race conditions can get fixed for the future. Glad it's solved :)